web/app/themes/ppj/src/sass/colors.sass

```diff
@@ -1,3 +1,27 @@
+$color-dark-transparent: rgba(0,0,0,0.85)
+$color-medium-transparent: rgba(0,0,0,0.3)
+$color-white: #fff
+$color-white-transparent: rgba(255,255,255,0.9)
+$color-transparent: rgba(0,0,0,0)
+$color-transparent2: rgba(255,255,255,0)
+$color-main-hover: #007197
+$color-secondary: rgb(30,147,189)
+$color-section-highlighted: rgb(240,229,239)
+$color-extremely-light: rgb(240,240,240)
+$color-very-light: rgb(229,229,232)
```

Comment:
Are these the same variables as used in the other PR(s)? This could get messy at merge time since all 3 branches contain similar stuff.

Reply:
Hopefully the merge is ok as this is from a cherry picked commit.